prepare_release.csx

```diff
@@ -38,7 +57,115 @@ var pr = await gitHubClient.PullRequest.Create(
     new NewPullRequest($"Release {version}", releaseBranchName, targetBranchName));
 Console.WriteLine($"Created pull request '{pr.Title}' at {pr.HtmlUrl}");
 
+public static GitHubClient GetAuthenticatedGitHubClient()
+{
+    var token = ReadGitHubToken();
+    var credentials = new Credentials(token);
+    return new GitHubClient(new ProductHeaderValue("FakeItEasy-build-scripts")) { Credentials = credentials };
+}
+
 public static string ReadGitHubToken()
 {
     return File.ReadAllText(".githubtoken").Trim();
+}
+
+public static async Task<Milestone> GetExistingMilestone()
+{
+    Console.WriteLine("Fetching milestone 'vNext'...");
+    var milestoneRequest = new MilestoneRequest { State = ItemStateFilter.Open };
+    var existingMilestone = (await gitHubClient.Issue.Milestone.GetAllForRepository(repoOwner, repoName, milestoneRequest))
+        .Single(milestone => milestone.Title == "vNext");

```

Comment:
This seems a bit fragile... but I can't think of a good alternative. Too bad we can't add arbitrary metadata to Github items.

Reply:
Yeah, I couldn't come up with anything better either.